src/winMain.py

```diff
@@ -0,0 +1,126 @@
+# from multiprocessing.connection import wait
+from PyQt5.QtWidgets import *
+from PyQt5 import uic
+from courses import *
+# import threading
+# import multiprocessing
+from datetime import datetime
+
+courses = {}
+# openCourse = []
+# waitlistCourse = []
+fullList = []
+fullDict = {}
+
+# TODO: MAKE THING SO THEY CAN REMOVE CRNs 
+
+class Ui(QDialog):
+    def __init__(self):
+        super(Ui, self).__init__() # Call the inherited classes __init__ method
+        uic.loadUi('src\qtWin.ui', self) # Load the .ui file
```

Comment:
would probably need to make a second file for unix/linux because file pathing is handled differently between windows and unix/linux